src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -635,8 +636,10 @@ function prepareQuestionForm(type) {
 
         $('#contribForm').show();
         fixContribQnGiverRecipient(NEW_QUESTION);
-        setDefaultContribQnVisibility(NEW_QUESTION);
         setContribQnVisibilityFormat(NEW_QUESTION);
+        if (!didCopyOptions) {
```

Comment:
There are some [exceptions](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#specific-naming-conventions) (`has`, `can`, `should`), but I acknowledge that `did` is not one of them. Will change to `has`.